packages/live-preview-sdk/src/csm/index.ts

```diff
@@ -0,0 +1,86 @@
+import { vercelStegaEncode } from '@vercel/stega';
+import { debug } from '../helpers';
+import jsonPointer from 'json-pointer';
+import { EntitySource, GraphQLResponse, Source } from './types';
+
+const isUrlOrIsoDate = (value: string) => {
```

Comment:
❓ couldn't we solve that later on with the content type?

Reply:
ah but also false positives as it is only appearance and still symbol in the background right?
then nvm